CompetitionTemplate/src/competition/subsystems/autonomous/LowBarScoreCommandGroup.java

```diff
@@ -0,0 +1,41 @@
+package competition.subsystems.autonomous;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import competition.subsystems.drive.DriveSubsystem;
+import competition.subsystems.drive.PoseSubsystem;
+import competition.subsystems.drive.commands.HeadingModule;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+@Singleton
+public class LowBarScoreCommandGroup extends CommandGroup{
+    public DoubleProperty distanceToTurningPoint;
+    public DoubleProperty distanceToLowGoal;
+    DoubleProperty targetHeading;
+    
+    @Inject
+    public LowBarScoreCommandGroup (DriveSubsystem driveSubsystem, HeadingModule headingModule, PoseSubsystem poseSubsystem, XPropertyManager propManager){
+        distanceToTurningPoint = propManager.createPersistentProperty("distance to turning point", 147.7);
+        DriveForDistanceCommand driveToTurningPoint = new DriveForDistanceCommand(poseSubsystem, driveSubsystem, propManager);
```

Comment:
Instead of creating a new DriveForDistanceCommand, you can ask for it in the constructor and it will inject one in for you